docs/LOCALIZATION.md

```diff
@@ -0,0 +1,248 @@
+| `he-IL`       | Hebrew                | עברית            |
+| `hi-IN`       | Hindi                 | हिंदी            |
+| `hu-HU`       | Hungarian             | magyar           |
+| `id-ID`       | Indonesian            | Bahasa Indonesia |
+| `it-IT`       | Italian               | italiano         |
+| `ja-JP`       | Japanese              | 日本語           |
+| `kk-KZ`       | Kazakh                | Қазақ            |
+| `ko-kr`       | Korean                | 한국어           |
+| `lv-LV`       | Latvian               | latviešu         |
+| `lt-LT`       | Lithuanian            | lietuvių         |
+| `ms-MY`       | Malay                 | Bahasa Melayu    |
+| `nb-NO`       | Norwegian (Bokmål)    | norsk (bokmål)   |
+| `pl-PL`       | Polish                | Polski           |
+| `pt-BR`       | Portuguese (Brazil)   | Português        |
+| `pt-PT`       | Portuguese (Portugal) | português        |
+| `ro-RO`       | Romanian              | română           |
+| `ru-RU`       | Russian               | Русский          |
+| `sr-Cyrl`     | Serbian (Cyrillic)    | српски           |
+| `sr-Latn`     | Serbian (Latin)       | srpski           |
+| `sk-SK`       | Slovak                | slovenčina       |
+| `sl-SI`       | Slovenian             | slovenski        |
+| `es-ES`       | Spanish               | español          |
+| `sv-SE`       | Swedish               | svenska          |
+| `th-TH`       | Thai                  | ไทย              |
+| `tr-TR`       | Turkish               | Türkçe           |
+| `uk-UA`       | Ukrainian             | українська       |
+| `vi-VN`       | Vietnamese            | Tiếng Việt       |
+
+### Community-contributed languages
+
+| Language Code | Language        | Name           | Translator                               |
+| ------------- | --------------- | -------------- | ---------------------------------------- |
+| `ar-EG`       | Egyptian Arabic | اللهجه المصريه | [@midineo](https://github.com/midineo)   |
+| `zh-HK`       | Cantonese       | 廣東話         | [@compulim](https://github.com/compulim) |
+
+### JSON file format
+
+This file format is maintained by localization team at Microsoft. An automated pipeline has been set up and will help Web Chat to translate new strings in a matter of days than weeks or months.
+
+### Lock down on strings
+
+For strings that are maintained by Microsoft, they have passed additional checks, for example, profanity check and cultural check, to guarantee the quality of the strings. In order to maintain the quality of these strings, modification of these strings requires additional checks from the localization team.
+
+The additional checks do not applies for strings that are contributed by the community, for example, Egyptian Arabic and Cantonese.
+
+### Overrideable by developers
+
+Developers should have a way to modify existing strings. One of the popular ask is the modify the "Type your message" placeholder text in the message input box.
```

Comment:
```suggestion
Developers may modify existing strings for customization purposes. One popular ask is the modification of the "Type your message" placeholder text in the message input box.
```
message input box => SendBox?

Reply:
I prefer to keep it as message input box, because in short future, we will have speech input box, and other input box. I can also name it "text input box".

What do you think?